### DISCUSSION TITLE
Automatically set the assignee on a new pull request

### DISCUSSION THREAD

**Discussion by ipocentro87** [magit/forge#536] [Ideas]
Hi all, I was wondering whether there is a way to automatically set some fields such as the assignee when a pull request is made. I am using Gitlab.
Thanks in advance!

**Comment by tarsius** [comment]
Not yet, no. Adding support isn't a priority but also not off the table.

**Reply by tarsius** [reply]
I do not intend to implement that.

This is the boilerplate you would need to implement this:

```elisp
(defun my-forge-create-pullreq-auto-assign ()
  (when (eq forge-edit-post-action 'new-pullreq)
    (setq forge--buffer-assignees
          (secret-sauce forge--buffer-base-branch
                        forge--buffer-head-branch))))

(add-hook 'forge-edit-post-hook
          #'my-forge-create-pullreq-auto-assign)
```

but you would have to implement the `secret-sauce` yourself.
